cmake/package_nsis.cmake

```diff
@@ -50,29 +50,44 @@ set(VC_REDIST_EXE "vcredist_${ARCH}.exe")
 ## Find redistributable to be installed by NSIS
 if (NOT VC_REDIST_PATH)
 	string(REGEX REPLACE ".*Visual Studio ([1-9][1-9]) .*" "\\1" OPENMS_MSVC_VERSION_STRING "${CMAKE_GENERATOR}")
-	if("${OPENMS_MSVC_VERSION_STRING}" GREATER "14")
-	  ## An old solution to find redists
-	  #execute_process(COMMAND "$ENV{PROGRAMFILES}/Microsoft Visual Studio/Installer/vswhere" -latest -version "${OPENMS_MSVC_VERSION_STRING}" -property installationPath
-	  #                OUTPUT_VARIABLE VC_ROOT_PATH
-	  #				  ERROR_VARIABLE VSWHERE_ERROR
-	  #				  RESULT_VARIABLE VSWHERE_RESULT)
-	  #if ("${VSWHERE_RESULT}" NOTEQUAL "0")
-	  #  message(FATAL_ERROR "Executing vswhere to find vsredist executable for win packaging failed. Either specify VC_REDIST_PATH or make sure vswhere works.")
-	  #endif()
-	  
-	  ## We have to glob recurse in the parent folder because there is a version number in the end.
-	  ## Unfortunately in my case the default version (latest) does not include the redist?!
-	  ## TODO Not sure if this environment variable always exists. In the VS command line it should! Fallback vswhere or VCINSTALLDIR/Redist/MSVC?
-	  get_filename_component(VC_ROOT_PATH "$ENV{VCToolsRedistDir}.." ABSOLUTE)
+	if("${OPENMS_MSVC_VERSION_STRING}" GREATER "17")
+	  ## according to https://docs.microsoft.com/de-de/cpp/windows/redistributing-visual-cpp-files?view=msvc-160
+	  get_filename_component(VC_ROOT_PATH "$ENV{VCINSTALLDIR}Redist/MSVC/v142" ABSOLUTE)
```

Comment:
Should we check that it is really in the env? For abibuilder8 it was not automatically in the System Env.

Reply:
Can you check on the VS command line? Or was it?

Reply:
But yes, checking is better in any case

Reply:
Could you do that for the next if, too?